install_files/ansible-base/roles/app-test/tasks/main.yml

```diff
@@ -14,3 +14,7 @@
 - include: dev_setup_xvfb_for_functional_tests.yml
 
 - include: disable_mprotect_ff_plugin_container.yml
+
+- include: download_verify_tbb.yml
+
+- include: download_ff_esr.yml
```

Comment:
@garrettr suggested dropping support for a fallback browser, and I agree especially because it would simplify the Ansible and Python here a moderate amount. Tor Browser or bust!